tests/cli/test_compose_ext4-filesystem.sh

```diff
@@ -16,6 +16,13 @@ set -e
 
 CLI="${CLI:-./src/bin/composer-cli}"
 
+if [[ -v SUB_HOSTNAME && -v SUB_USER && -v SUB_PASS ]]; then
```

Comment:
Change to `SUBSCRIPTION_`, no need to be terse when you can be verbose.

Also make sure you are actually using variables, e.g. `$SUBSCRIPTION_HOSTNAME`.

What does `-v` do? I can't find it documented and I've never seen it before?

Reply:
Maybe also `rlLogInfo` because I can't tell if the current test execution used subscriptions or not.

Reply:
> What does `-v` do? I can't find it documented and I've never seen it before?
I found it here https://www.tutorialkart.com/bash-shell-scripting/check-if-variable-is-set/  It works but like you said it does not seem to be documented.  I'll change it to `-z` or `-n`